shed/tests/test_translation.py

```diff
@@ -406,3 +407,17 @@ def test_dbfriendly(RE, hw):
     h2 = d[0].get("graph_hash")
     assert h1 != h2
     assert len(d) == 10 + 3
+
+
+def test_db_insertion(RE, hw):
+    db = Broker.named("temp")
```

Comment:
Does this fail on master branch?

Reply:
I test this in the jupyter notebook. I failed because of the float number keys.